tooltester/src/main/resources/PipelineTest.groovy

```diff
@@ -1,4 +1,4 @@
-currentBuild.displayName = "1.3.0-beta.1"
+currentBuild.displayName = "1.4.0-beta.3"
```

Comment:
This version seems odd

Reply:
Groovy script in general is mostly a back-up in case Jenkins explodes.  I've been changing it manually between releases.